src/vaadin-details.js

```diff
@@ -136,6 +137,7 @@ class DetailsElement extends ControlStateMixin(ElementMixin(ThemableMixin(Polyme
 
   ready() {
     super.ready();
+    this._contentId = this._getContentId();
```

Comment:
Please align this with the implementation of [`vaadin-radio-group`](https://github.com/vaadin/vaadin-radio-button/blob/ac80d5e6cac1b266c44da81bac79a1c13e930d15/src/vaadin-radio-group.js#L263-L264):

```js
    const uniqueId = (DetailsElement._uniqueId = 1 + DetailsElement._uniqueId || 0);
    this. _contentId = `${this.constructor.is}-content-${uniqueId}`;
```

Note the `DetailsElement` is defined in this file so you don't need to rely on `Vaadin.registrations`.